apps/ensrainbow/src/ingest.ts

```diff
@@ -57,8 +63,7 @@ async function loadEnsNamesToLevelDB(): Promise<void> {
         const [hashVal, name] = parts;
         if (hashVal && name) {
```

Comment:
Following `isHex` usage from the other file on this PR:
```suggestion
        if (isHex(hashVal) && name) {
```